server/src/lib/model/db/migrations/20191017135015-add-challenge-team-to-user-clients.ts

```diff
@@ -0,0 +1,12 @@
+export const up = async function(db: any): Promise<any> {
+  return db.runSql(
+    `
+    ALTER TABLE user_clients ADD COLUMN challenge_team VARCHAR(255) DEFAULT NULL;
+    ALTER TABLE user_clients ADD CONSTRAINT uc_ibfk_1 FOREIGN KEY (challenge_team) REFERENCES teams(team_name);
```

Comment:
This looks fine to me, but I just want to run a quick confidence check. Will the default `NULL` value pass this constraint? Just want to make sure, since this will run across the entire `user_clients` table.

Reply:
(Update: tested it on my end, it works!)